include/boost/detail/winapi/dbghelp.hpp

```diff
@@ -0,0 +1,167 @@
+typedef CHAR_ 		 *PSTR_;
+
+
+struct IMAGE_DATA_DIRECTORY_ {
+  DWORD_ VirtualAddress;
+  DWORD_ Size;
+};
+
+struct IMAGE_EXPORT_DIRECTORY_ {
+  DWORD_ Characteristics;
+  DWORD_ TimeDateStamp;
+  WORD_ MajorVersion;
+  WORD_ MinorVersion;
+  DWORD_ Name;
+  DWORD_ Base;
+  DWORD_ NumberOfFunctions;
+  DWORD_ NumberOfNames;
+  DWORD_ AddressOfFunctions;
+  DWORD_ AddressOfNames;
+  DWORD_ AddressOfNameOrdinals;
+};
+
+struct IMAGE_FILE_HEADER_ {
+  WORD_ Machine;
+  WORD_ NumberOfSections;
+  DWORD_ TimeDateStamp;
+  DWORD_ PointerToSymbolTable;
+  DWORD_ NumberOfSymbols;
+  WORD_ SizeOfOptionalHeader;
+  WORD_ Characteristics;
+} ;
+
+
+struct IMAGE_OPTIONAL_HEADER32_ {
+
+  WORD_ Magic;
+  BYTE_ MajorLinkerVersion;
+  BYTE_ MinorLinkerVersion;
+  DWORD_ SizeOfCode;
+  DWORD_ SizeOfInitializedData;
+  DWORD_ SizeOfUninitializedData;
+  DWORD_ AddressOfEntryPoint;
+  DWORD_ BaseOfCode;
+  DWORD_ BaseOfData;
+  DWORD_ ImageBase;
+  DWORD_ SectionAlignment;
+  DWORD_ FileAlignment;
+  WORD_ MajorOperatingSystemVersion;
+  WORD_ MinorOperatingSystemVersion;
+  WORD_ MajorImageVersion;
+  WORD_ MinorImageVersion;
+  WORD_ MajorSubsystemVersion;
+  WORD_ MinorSubsystemVersion;
+  DWORD_ Win32VersionValue;
+  DWORD_ SizeOfImage;
+  DWORD_ SizeOfHeaders;
+  DWORD_ CheckSum;
+  WORD_ Subsystem;
+  WORD_ DllCharacteristics;
+  DWORD_ SizeOfStackReserve;
+  DWORD_ SizeOfStackCommit;
+  DWORD_ SizeOfHeapReserve;
+  DWORD_ SizeOfHeapCommit;
+  DWORD_ LoaderFlags;
+  DWORD_ NumberOfRvaAndSizes;
+  IMAGE_DATA_DIRECTORY_ DataDirectory[16];
+};
+
+struct IMAGE_OPTIONAL_HEADER64_ {
+  WORD_ Magic;
+  BYTE_ MajorLinkerVersion;
+  BYTE_ MinorLinkerVersion;
+  DWORD_ SizeOfCode;
+  DWORD_ SizeOfInitializedData;
+  DWORD_ SizeOfUninitializedData;
+  DWORD_ AddressOfEntryPoint;
+  DWORD_ BaseOfCode;
+  ULONGLONG_ ImageBase;
+  DWORD_ SectionAlignment;
+  DWORD_ FileAlignment;
+  WORD_ MajorOperatingSystemVersion;
+  WORD_ MinorOperatingSystemVersion;
+  WORD_ MajorImageVersion;
+  WORD_ MinorImageVersion;
+  WORD_ MajorSubsystemVersion;
+  WORD_ MinorSubsystemVersion;
+  DWORD_ Win32VersionValue;
+  DWORD_ SizeOfImage;
+  DWORD_ SizeOfHeaders;
+  DWORD_ CheckSum;
+  WORD_ Subsystem;
+  WORD_ DllCharacteristics;
+  ULONGLONG_ SizeOfStackReserve;
+  ULONGLONG_ SizeOfStackCommit;
+  ULONGLONG_ SizeOfHeapReserve;
+  ULONGLONG_ SizeOfHeapCommit;
+  DWORD_ LoaderFlags;
+  DWORD_ NumberOfRvaAndSizes;
+  IMAGE_DATA_DIRECTORY_ DataDirectory[16];
+};
+
+
+struct IMAGE_NT_HEADERS64_ {
+  DWORD_ Signature;
+  IMAGE_FILE_HEADER_ FileHeader;
+  IMAGE_OPTIONAL_HEADER64_ OptionalHeader;
+};
+
+struct IMAGE_NT_HEADERS32_ {
+  DWORD_ Signature;
+  IMAGE_FILE_HEADER_ FileHeader;
+  IMAGE_OPTIONAL_HEADER32_ OptionalHeader;
+};
+
+
+#ifdef _WIN64
+    typedef IMAGE_NT_HEADERS64_ IMAGE_NT_HEADERS_;
+#else  /* _WIN64 */
+    typedef IMAGE_NT_HEADERS32_ IMAGE_NT_HEADERS_;
+#endif /* _WIN64 */
+
+
+__declspec(dllimport) IMAGE_NT_HEADERS_*  WINAPI ImageNtHeader(PVOID_ Base);
```

Comment:
Oh, thanks for reminding me. Actually no, because you have all that stuff in pe_info.hpp. That was inherited from my dll project, before I started on inserting it into boost.dll.